src/System.Private.ServiceModel/tests/Common/Unit/UnitTests.Common.csproj

```diff
@@ -7,23 +7,21 @@
   <PropertyGroup>
     <MinimumVisualStudioVersion>11.0</MinimumVisualStudioVersion>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>UnitTests.Common</RootNamespace>
     <AssemblyName>UnitTests.Common</AssemblyName>
     <SignAssembly>false</SignAssembly>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ProjectGuid>{E896294A-AB4A-4AF5-A01C-A19E3972EFF9}</ProjectGuid>
-    <IsTestProject>true</IsTestProject>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="**\*.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(WcfDuplexPkgProj)" />
-    <ProjectReference Include="$(WcfHttpPkgProj)" />
-    <ProjectReference Include="$(WcfPrimitivesPkgProj)" />
+    <ProjectReference Include="$(WcfDuplexRefProj)" />
+    <ProjectReference Include="$(WcfHttpRefProj)" />
+    <ProjectReference Include="$(WcfPrimitivesRefProj)" />
```

Comment:
This seems reasonable, but I can't find any precedent. Test projects in CoreFX just reference each other, never back to the product.